contents/customers/hostai.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How HostAI prevents churn with PostHog and LangFuse
```

Comment:
```suggestion
title: How HostAI uses PostHog and LangFuse to prevent churn
```

Not a strong feeling, just think it flows a bit better with us in an active role